app/controllers/prior_authority/steps/base_controller.rb

```diff
@@ -22,6 +22,11 @@ def as
       def service
         Providers::Gatekeeper::PAA
       end
+
+      def prune_navigation_stack
```

Comment:
would be good to get ride of this entirely.. but that will have to wait until we redo CRM7 task list